src/domain/Forms/VerticalForm/subcomponents/Field.tsx

```diff
@@ -23,6 +23,8 @@ interface IVerticalFormFieldProps {
   isRequired?: boolean
   /** Text displayed above the input */
   label?: string
+  /** Arbitrary text displayed after the label */
```

Comment:
Could mention here if it goes before or after the required label if that's active?